inspectit-oce-demo/agentconfig/api-gateway/config.yml

```diff
@@ -0,0 +1,176 @@
+#inspectit:
```

Comment:
hmm, is it okey to have a full config? I mean this represents then a very difficult point for maintaining.. Every time we change a default config file we would need to change this as well?

One solution could be to use symbolic links to the default config, but this is not working as well as the stuff would not be completely commented-out. 

What about if we just have a file with some `# comment` explaining what to do. I am also fine to create an empty hidden file (`.empty`) in these folders, so that at least folder structure is created correctly.

Reply:
The config file is completely commented out! So it's like an empty file.
It's also how other tools are doing this, so the user at least sees all the configuration options.